BTSerial/MainPage.xaml.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+using Windows.Devices.Bluetooth;
+using Windows.Devices.Bluetooth.Rfcomm;
+using Windows.Networking.Sockets;
+using Windows.Storage.Streams;
+using System.Threading.Tasks;
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace BTSerial
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+
+        // The Chat Server's custom service Uuid: 34B1CF4D-1069-4AD6-89B6-E161D79BE4D8
+
+        private Windows.Devices.Enumeration.DeviceInformationCollection deviceCollection;
+        private Windows.Devices.Enumeration.DeviceInformation selectedDevice;
+        Windows.Devices.Bluetooth.Rfcomm.RfcommDeviceService deviceService;
```

Comment:
Looks like you aren't using this. You are using local variable with the same name. You may want to remove the member variable, or remove the local variable instead to avoid confusion. If you choose the 2nd option, add 'private' qualifier.